docs/TUTORIAL.md

```diff
@@ -376,10 +369,12 @@ the target filepaths to metadata.
 # (octal number specifying file access for owner, group, others e.g., 0755) is
 # added alongside the default fileinfo.  All target objects in metadata include
 # the target's filepath, hash, and length.
+# Note: target path passed to add_target() method has to be relative
+# to the targets directory or an exception is raised.
 >>> target4_filepath = os.path.abspath("repository/targets/myproject/file4.txt")
 >>> octal_file_permissions = oct(os.stat(target4_filepath).st_mode)[4:]
 >>> custom_file_permissions = {'file_permissions': octal_file_permissions}
->>> repository.targets.add_target(target4_filepath, custom_file_permissions)
+>>> repository.targets.add_target('myproject/file4.txt', custom_file_permissions)
```

Comment:
Is it just me or does this pattern of assigning the full path to a variable and then passing the relative sub-path as a string look a little strange?

Reply:
Agree, it looks awkward but it is also a good example why a normalising path helper is needed :) 
I tired to rewrite it in a more reasonable way: https://github.com/theupdateframework/tuf/pull/1008/commits/6d8a84d9938d149e01c3bd0bfb52d056bb5b7b55